backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/organization/impl/OrgDAO.java

```diff
@@ -263,4 +263,10 @@ public ZoneId getTimeZone(@NonNull final OrgId orgId)
 		return timeZone != null ? timeZone : SystemTime.zoneId();
 	}
 
+	@Override
+	public boolean isEUOneStopShop(@NonNull final OrgId orgId)
+	{
+		return getById(orgId).isEUOneStopShop();
```

Comment:
getById might return null

Reply:
changed to throw exception if no org found.